### PR TITLE
fix(frontend): use static sitemap

### PR DESCRIPTION
**Pull request by lionelB**
use static site host on azure

### CODE REVIEW

File: packages/code-du-travail-frontend/next.config.js [hunk 1]
```diff
@@ -42,13 +42,7 @@ const nextConfig = {
     SENTRY_PUBLIC_DSN: process.env.SENTRY_PUBLIC_DSN,
   },
   async redirects() {
-    return [
-      {
-        destination: "/api/sitemap",
-        permanent: false,
-        source: "/sitemap.xml",
-      },
-    ];
```
Comment (revolunet) [inline packages/code-du-travail-frontend/next.config.js]: on peut garder ca par convention ?

> The location of a Sitemap file determines the set of URLs that can be included in that Sitemap. A Sitemap file located at http://example.com/catalog/sitemap.xml can include any URLs starting with http://example.com/catalog/ but can not include URLs starting with http://example.com/images/.

from https://www.sitemaps.org/protocol.html

File: packages/code-du-travail-frontend/server/koaServer.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ const robotsProd = [
   "Disallow: /assets/",
   "Disallow: /images/",
   "",
-  `Sitemap: https://${PROD_HOSTNAME}/sitemap.xml`,
+  `Sitemap: https://${PROD_HOSTNAME}/api/sitemap`,
```
Comment (revolunet) [inline packages/code-du-travail-frontend/server/koaServer.js]: a priori il faut le laisse à la racine

**Review by UnbearableBear — APPROVED** [review]
Looks good ! Il y a un moment ou on n'est pas en https ?

**Comment by lionelB** [comment]
> Looks good ! Il y a un moment ou on n'est pas en https ?

normalement non =)

**Pull request merged**